INSTALL.md

```diff
@@ -31,6 +31,8 @@ CONFIG_HAVE_BPF_JIT=y
 CONFIG_BPF_EVENTS=y
 ```
 
+For Linux kernel version 4.7 or newer, the flag `CONFIG_HAVE_BPF_JIT` has been replaced by `CONFIG_HAVE_EBPF_JIT` (for x86, arm64 and s390) or by `CONFIG_HAVE_CBPF_JIT` (for arm, mips, powerpc and sparc.)
```

Comment:
'misinformation' is my problem with this change.
re: 1) cbpf is always supported. ebpf is supported when CONFIG_BPF_SYSCALL is on
(and that bit is already covered by this INSTALL.md)
re: 2) CONFIG_HAVE_EBPF_JIT and CONFIG_HAVE_CBPF_JIT has nothing to do with kernel providing either cbpf or ebpf api to user space. These config flags are hidden and not selectable by users. CONFIG_BPF_JIT is a different flag that is selectable and it's on by default in all distros. Depending on (c|e)bpf jit availability in the architecture it will
improve performance of one or both types of programs, but doesn't affect functionality
and availability of JITs is different with kernel releases.
So I don't think there is a need to mention these info in this install doc.